scripts/test.py

```diff
@@ -41,18 +41,26 @@ def monte_carlo_sampling_chaospy(N_FEATURES, SAMPLES, rule="latin_hypercube", se
     """
     import chaospy
     from scipy.stats import qmc 
+    from sklearn.preprocessing import MinMaxScaler
+
 
     # Generate a Nfeatures-dimensional latin hypercube varying between 0 and 1:
-    N_FEATURES = "chaospy.Uniform(0, 1), "*N_FEATURES
+    N_FEATURES = f"chaospy.{DISTRIBUTION}(0, 1), "*N_FEATURES
     uniform_cube = eval(f"chaospy.J({N_FEATURES})")  # writes Nfeatures times the chaospy.uniform... command)
     lh = uniform_cube.sample(SAMPLES, rule=rule, seed=seed).T
+
+    # rescale distribution to 0-1 if it is not a uniform distribution
+    if DISTRIBUTION != "Uniform":
+        mm = MinMaxScaler()
+        lh = mm.fit_transform(lh)
```

Comment:
Agree with @pz-max 's suggestion from our discussion:

```python
if DISTRIBUTION == "Uniform":
    pass
elif DISTRIBUTION == "NormalDistribution":
    mm = MinMaxScaler()
    lh = mm.fit_transform(lh)
else:
    raise ValueError(f"Distribution '{DISTRIBUTION}' not available. Please pick a valid one from possible options: 'Uniform', 'NormalDistribution'")
```
Obviously, feel free to revise the suggestion. The point is, we can't be sure that normal distribution will be an appropriate default for any occasion. So, better to request clarification, if the input is not clear.

Reply:
This is a better approach. There is still going to be an improvement to add more options. Using an if else loop make it easier.

Reply:
> This is a better approach. There is still going to be an improvement to add more options. Using an if else loop make it easier.

Great! I think, looks very good now :) I also like `Normal` as a keyword for and argument. A very little comment: could you please adjust the error message, replacing `'NormalDistribution'` with `'Normal'` to match with the changes you suggested?